mrmustard/physics/ansatze.py

```diff
@@ -294,7 +294,7 @@ def c(self) -> Batch[ComplexTensor]:
         r"""
         The array of coefficients for the polynomial terms in the ansatz.
         """
-        return self.array
+        return self.c
```

Comment:
doesn't this create an infinite recursion?

Reply:
But I understand that the term "array" is better suited for the ArrayAnsatz. Perhaps we can change array to coeff in the PolyExp ansatz.